packages/api/controllers/hackathon_register_controller.go

```diff
@@ -51,13 +51,9 @@ func RegisterTeamMember(c *fiber.Ctx) error {
 		ShareInfoWithSponsors:          member.ShareInfoWithSponsors,
 		WantJobOffers:                  member.WantJobOffers}
 
-	numberOfTotalParticipants := GetTotalNumberOfParticipants()
+	// numberOfTotalParticipants := GetTotalNumberOfParticipants()
 	numberOfTeams, _ := GetNumberOfHackathonTeams(ctx)
 
-	if numberOfTotalParticipants >= 90 {
```

Comment:
You removed this because it's expected they won't be able to hit this endpoint because the form is hidden (e.g. request to `/available` endpoint returned 409 and we've disabled it), right? Well, that's a good thing you paid attention to that, unfortunately, we still need this within the `/register` endpoint. Why? Because these people are supposed to be devs and they might try to register for the hackathon by hitting the endpoint directly. This way we allow them to register even if the cap is reached when using for example Postman, but not our website.

Reply:
Yes. But I didn't think about protecting us from our own tribe